tools/tsdb-index/main.go

```diff
@@ -40,6 +45,17 @@ func main() {
 		fmt.Println("No block directory specified.")
 		return
 	}
+	ctx := context.Background()
+	if *showLabelValues {
+		for _, blockDir := range args {
+			err = printLabelValues(ctx, blockDir)
+			if err != nil {
+				fmt.Fprintln(os.Stderr, err.Error())
+				os.Exit(1)
+			}
+		}
+		return
+	}
```

Comment:
This is not extension of existing functionality, but completely different one ;)

Reply:
i think the name `tsdb-index` mislead me. I will rename the existing tool, update docs, and move the new code into `tsdb-labels`